components/measurement/SummaryText.js

```diff
@@ -16,7 +15,7 @@ const SummaryText = ({
 }) => {
   const intl = useIntl()
   const metadata = getTestMetadata(testName)
-  const formattedDateTime = intl.formatDate(moment.utc(date).toDate(), {
+  const formattedDateTime = intl.formatDate(date, {
```

Comment:
I think you need to cast this to `utc` otherwise `intl.formatDate` will assume it's in the default locale of the browser resulting and then when it converts it back to UTC it will be offset by the wrong value.

As a demonstration of this see: https://explorer-git-fork-majakomel-replace-momentjs-ooni1.vercel.app/measurement/20220211T235940Z_webconnectivity_DE_8881_n1_kS8lfHqxOpVopvoi?input=https%3A%2F%2Ftheintercept.com%2F

Where it says:
```
On February 11, 2022, 10:59 PM UTC
```

yet the test_start_time value (expressed in UTC) is `"2022-02-11 23:59:39"`

Reply:
React Intl library was actually supposed to handle that (see https://github.com/ooni/explorer/blob/b47038a930da7a9fc7f31110999a748669ad5feb/components/measurement/SummaryText.js#L24) and I haven't noticed any problems with it locally (the link above doesn't work so couldn't check there). I ended up replacing those too with dayJS ( also because `react-intl` is currenly 2 major versions behind).